src/assessments/seo/TextLengthAssessment.js

```diff
@@ -90,6 +92,20 @@ export default class TextLengthAssessment extends Assessment {
 		return null;
 	}
 
+	/**
+	 * Checks if the feedback should include "slightly".
+	 *
+	 * @param {number} score The score the text received for the assessment.
+	 *
+	 * @returns {boolean} Whether the feedback should include "slightly".
+	 */
+	useSlightly( score ) {
```

Comment:
This function combines two things:
- Checking if the score is slightly below the minimum.
- If this is cornerstone content.

I think it is better to have purer functions to avoid future confusion.
This will also result in a clearer name of the function. I don't think the current name covers it. Also, if the score is not slightly below minimum, it should return false, but it does not return anything specific (thus undefined).

I would suggest `isCornerstoneContent` and `isSlightlyBelowMinimum` as two separate functions.